### PR TITLE
Tablet mode

### PR DESCRIPTION
**Pull request by benrudolph**
@biyeun @wpride this gives us tablet mode! this commit was cherry picked: https://github.com/dimagi/commcare-hq/commit/ed4d712f4b39b97db3a56ac265be9fa5f5dd6a2a
not quite perfect, but a good starting point. again, UI/UX to be dealt with later

🐟 or 🏠 

giphy

cc: @mkangia

### CODE REVIEW

File: corehq/apps/style/static/preview_app/less/preview_app/variables.less [hunk 1]
```diff
@@ -0,0 +1,2 @@
+@preview-phone-width: 250px;
+@preview-tablet-width: 450px;
```
Comment (benrudolph) [inline corehq/apps/style/static/preview_app/less/preview_app/variables.less]: @biyeun i redefined these here, not sure if you have any thoughts as to whether or not that makes sense or not

Comment (biyeun) [inline corehq/apps/style/static/preview_app/less/preview_app/variables.less]: i would only have them defined in one place. `variables.less` seems good

**Review by biyeun — APPROVED** [review]
other than duplicating the variables in `variables.less` seems good to me :)

**Comment by biyeun** [comment]
waiting on ✅

File: corehq/apps/cloudcare/static/cloudcare/js/formplayer/middleware.js [hunk 2]
```diff
@@ -28,10 +28,24 @@ FormplayerFrontend.module("SessionNavigate", function (SessionNavigate, Formplay
...
+            maxHeightForm;
+        maxHeight = ($(window).height() -
+            FormplayerFrontend.regions.phoneModeNavigation.$el.height() -
+            FormplayerFrontend.regions.breadcrumb.$el.height());
+        maxHeightForm = ($(window).height() -
+            FormplayerFrontend.regions.phoneModeNavigation.$el.height());
+
+        $('.scrollable-container').css('max-height', maxHeight + 'px');
+        $('.form-scrollable-container').css('min-height', maxHeightForm + 'px');
+        $('.form-scrollable-container').css('max-height', maxHeightForm + 'px');
```
Comment (mkangia) [inline corehq/apps/cloudcare/static/cloudcare/js/formplayer/middleware.js]: @benrudolph is it better to have repeating DOM fetch as an object?
```
var scrollable_container = $('.form-scrollable-container')
scrollable_container.css(....)
scrollable_container.css(....)
```
Or chaining in such case is also a good way out?

**Comment by benrudolph** [comment]
updated, @biyeun will take care of variable dryness in followup PR

**Pull request merged**